kukkuu/urls.py

```diff
@@ -24,3 +25,4 @@ def readiness(*args, **kwargs):
 
 
 urlpatterns += [path("healthz", healthz), path("readiness", readiness)]
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
This should be used only when `DEBUG` is `True`.

Reply:
Aw shet, yeah,  fixed it